### DISCUSSION TITLE
Hover instead of click to open dropdown

### DISCUSSION THREAD

**Discussion by SebGbs** [bootscore/bootscore#679] [Q&A]
Hello!
I've copied the class-bootstrap-5-navwalker.php from bootscore parent into the child theme aswell as copied the part of the functions.php from parent regarding reqiure_once the file. I've edited the file to create a megamenu which works fine, but I need the dropdowns to be opened by hover instead of click - is this possible by editing the file or is it css or js based?

Thanks in advance!

**Comment by crftwrk** [comment]
Please check here https://github.com/bootscore/bootscore/discussions/224

**Reply by SebGbs** [reply]
Hello crftwrk
I've noticed that an "show" class is being added to the dropdown-menu aswell as the parent. There is all some css-rules added:
`.dropdown-menu[data-bs-popper] {
    top: 100%;
    left: 0;
    margin-top: var(--bs-dropdown-spacer);
}`
These don't get added with the css-solution, do they?

**Reply by crftwrk** [reply]
Hi Sebastian

> These don't get added with the css-solution, do they?

These don't get added. Bootstrap uses popper.js for dropdowns, which adds these rules on click. Open dropdowns on hover means that dropdowns are always open but invisible if toggler is not hovered. So, popper.js is not in action here, because it needs a click.

BTW, https://bootstrap-menu.com is a great resource with code examples if you want to extend Bootstrap's default navbar component.
